File: Project_Euler/JavaScript/Problem2.js
var a=1, b=1, sum=0;
while(a + b < 4000000){
    a = a + b
    if(a % 2 == 0){
	sum += a
    }
    b = a + b
    if(b % 2 == 0){
	sum += b
    }
}
console.log(sum)

File: Project_Euler/Rust/Problem2.rs
fn main() {
    let mut a = 0;
    let mut b = 1;
    let mut sum = 0;
    while a < 4000000 && b < 4000000 {
        a += b;
        if a % 2 == 0 {
            sum += a
        }
        b += a;
        if b % 2 == 0 {
            sum += b
        }
    }
    print! {"{}",sum};
}
